sh/deploy_new_settler.sh

```diff
@@ -140,22 +140,7 @@ fi
 
 . "$project_root"/sh/common_safe_deployer.sh
 . "$project_root"/sh/common_deploy_settler.sh
-
-# set minimum gas price to (mostly for Arbitrum and BNB)
-declare -i min_gas_price
-min_gas_price="$(get_config minGasPriceGwei)"
-min_gas_price=$((min_gas_price * 1000000000))
-declare -r -i min_gas_price
-declare -i gas_price
-gas_price="$(cast gas-price --rpc-url "$rpc_url")"
-if (( gas_price < min_gas_price )) ; then
-    echo 'Setting gas price to minimum of '$((min_gas_price / 1000000000))' gwei' >&2
-    gas_price=$min_gas_price
-fi
-declare -r -i gas_price
-declare -i gas_estimate_multiplier
-gas_estimate_multiplier="$(get_config gasMultiplierPercent)"
-declare -r -i gas_estimate_multiplier
+. "$project_root"/sh/common_gas.sh
```

Comment:
Further down (around line 214) you need to replace the existing multiplier logic with the DRY'd logic from `common_gas.sh`